tests/src/podman-machine-onboarding.spec.ts

```diff
@@ -0,0 +1,192 @@
+      await playExpect(dashboardPage.mainPage).toBeVisible();
+      await playExpect(dashboardPage.notificationsBox).toBeVisible();
+      notificationPodmanSetup = dashboardPage.notificationsBox
+        .getByRole('region')
+        .filter({ hasText: 'Podman needs to be set up' });
+      await playExpect(notificationPodmanSetup).toBeVisible();
+    });
+    describe('Onboarding navigation', async () => {
+      test('Open Podman Machine Onboarding through Setup Notification', async () => {
+        await notificationPodmanSetup.getByTitle('Set up Podman').click();
+        podmanOnboardingPage = await checkPodmanMachineOnboardingPage(page);
+      });
+      test('Return to Dashboard', async () => {
+        dashboardPage = await navigationBar.openDashboard();
+        await playExpect(dashboardPage.mainPage).toBeVisible();
+      });
+      test('Re-Open Podman Machine Onboarding through Settings Resources page', async () => {
+        settingsBar = await navigationBar.openSettings();
+        await settingsBar.resourcesTab.click();
+        resourcesPage = new ResourcesPage(page);
+        await playExpect(resourcesPage.podmanResources).toBeVisible();
+        await resourcesPage.podmanResources.getByLabel('Setup Podman').click();
+        podmanOnboardingPage = await checkPodmanMachineOnboardingPage(page);
+      });
+    });
+    test('Verify Podman Autostart is enabled and proceed to next page', async () => {
+      await playExpect(podmanOnboardingPage.podmanAutostartToggle).toBeChecked();
+      await podmanOnboardingPage.nextStepButton.click();
+    });
+    test('Expect no machine created message and proceed to next page', async () => {
+      await playExpect(podmanOnboardingPage.onboardingStatusMessage).toHaveText(
+        `We could not find any Podman machine. Let's create one!`,
+      );
+      await podmanOnboardingPage.nextStepButton.click();
+    });
+    test('Verify default podman machine settings', async () => {
+      await playExpect(podmanOnboardingPage.createMachinePageTitle).toHaveText(`Create a Podman machine`);
+      await playExpect(podmanOnboardingPage.podmanMachineConfiguration).toBeVisible();
+      await playExpect(podmanOnboardingPage.podmanMachineName).toHaveValue('podman-machine-default');
+      await playExpect(podmanOnboardingPage.podmanMachineCPUs).toBeVisible();
+      await playExpect(podmanOnboardingPage.podmanMachineMemory).toBeVisible();
+      await playExpect(podmanOnboardingPage.podmanMachineDiskSize).toBeVisible();
+      await playExpect(podmanOnboardingPage.podmanMachineImage).toHaveValue('');
+      await playExpect(podmanOnboardingPage.podmanMachineRootfulCheckbox).toBeChecked();
+      await playExpect(podmanOnboardingPage.podmanMachineUserModeNetworkingCheckbox).not.toBeChecked();
+      await playExpect(podmanOnboardingPage.podmanMachineStartAfterCreationCheckbox).toBeChecked();
+    });
+  });
+  describe('Podman Machine creation and operations', async () => {
```

Comment:
Add condition to run this test, ie. TEST_PODMAN_MACHINE. Same would be used as a param, if the clean up will happen in beforeall

Reply:
done.